kaggle_environments/envs/halite/halite.py

```diff
@@ -87,9 +87,12 @@ def populate_board(state, env):
     uid_counter = 0
 
     # Set seed for random number generators
-    if hasattr(config, "randomSeed"):
-        np.random.seed(config.randomSeed)
-        seed(config.randomSeed)
+    if not hasattr(config, "randomSeed"):
```

Comment:
Little confused about this logic flip, just wanted to draw attention in case it wasn't intentional

Reply:
This is for item 3 in the description -- previously we would read the halite seed if it was in the configuration but we would never set a seed. Now if there is not a seed set we always set one and we always read the seed from configuration. This means that for production episodes you'll now also get the seed in the replay file.